src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/BaseConfiguration.java

```diff
@@ -471,9 +471,13 @@ private void initTagletManager(Set<List<String>> customTagStrs) {
                     tagletManager.addCustomTag(args.get(1), fileManager);
                     continue;
                 }
-                List<String> tokens = tokenize(args.get(1), TagletManager.SIMPLE_TAGLET_OPT_SEPARATOR, 3);
+                /* Since there are few constraints on the characters in a tag name,
+                 * and real world examples with ':' in the tag name, we cannot simply use
+                 * String.split(regex);  instead, we tokenize the string, allowing
+                 * special characters to be escaped with '\'. */
+                List<String> tokens = tokenize(args.get(1),3);
```

Comment:
This should have a space:
```suggestion
                List<String> tokens = tokenize(args.get(1), 3);
```

Reply:
Fixed.